src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -38,8 +48,9 @@ internal class CodeWriter constructor(
   out: Appendable,
   private val indent: String = DEFAULT_INDENT,
   private val memberImports: Map<String, Import> = emptyMap(),
-  val importedTypes: Map<String, ClassName> = emptyMap()
-) {
+  val importedTypes: Map<String, ClassName> = emptyMap(),
+  private val wrapWhitespace: Boolean = false
```

Comment:
also changed most toString() methods to turn off line-wrap. It’s only on when you emit an entire file, which feels appropriate.